chart/compass/templates/rewrite-client-cert-filter.yaml

```diff
@@ -21,7 +21,8 @@ spec:
         operation: INSERT_BEFORE
         value:
           name: envoy.lua
-          config:
+          typed_config:
```

Comment:
it won't simply work this way. Compass repo is still running on kyma version with istio 1.4.